src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.cc

```diff
@@ -424,7 +429,41 @@ BtreeLBAManager::BtreeLBAManager(
   SegmentManager &segment_manager,
   Cache &cache)
   : segment_manager(segment_manager),
-    cache(cache) {}
+    cache(cache)
+{
+  register_metrics();
+}
+
+LBABtree::lba_tree_inner_stats_t LBABtree::lba_tree_inner_stats;
+void BtreeLBAManager::register_metrics()
+{
+  namespace sm = seastar::metrics;
+  auto tree_label = sm::label("tree");
+  auto lba_label = tree_label("LBA");
+  metrics.add_group(
+    "LBA",
+    {
+      sm::make_counter(
+        "lba_tree_alloc_extents",
+        LBABtree::lba_tree_inner_stats.num_alloc_extents,
+        sm::description("total number of lba alloc_extent operations"),
+        {lba_label}
+      ),
+      sm::make_counter(
+        "lba_tree_iter_prevs",
+        LBABtree::lba_tree_inner_stats.num_iter_prevs,
+        sm::description("total number of lba iter_prev operations"),
+        {lba_label}
+      ),
+      sm::make_counter(
+        "lba_tree_iter_nexts",
```

Comment:
It's more accurate to be called, for example, "alloc_extents_iter_nexts", with the description "total number of iterator next operations during extent allocation"